<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonBisection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Bisection</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFixedPoint">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fake Point</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_e1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>21</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-10</number>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>-5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNewtonRaphson">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>NewtonRaphson</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_e1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>E1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_variavelA_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string> A:        MS to LS </string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_variableA_0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>16</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_e2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>21</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-10</number>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>-5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_e1_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>E2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_maxIter">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Iter</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_maxIter">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>16</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_limiteInferior">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>69</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_limiteSuperior">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>69</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>3.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_from">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_to">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_outVariableA_0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_outE1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_outE2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_outMaxIter">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_outE1_10">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>10^</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_outE2_10">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>10^</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_variableA_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>16</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_outVariableA_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_variableA_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>16</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_outVariableA_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_variableA_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>16</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_outVariableA_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="virgula">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>454</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNumerical_Methods">
    <property name="title">
     <string>Numerical Methods</string>
    </property>
   </widget>
   <addaction name="menuNumerical_Methods"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
